AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Models/Options/AWSAuthConfirmResetPasswordOptions.swift

```diff
@@ -10,4 +10,8 @@ import Foundation
 public struct AWSAuthConfirmResetPasswordOptions {
 
     public let metadata: [String: String]?
+
+    public init(metadata: [String: String]?) {
```

Comment:
Give a default value as nil here and through out, so that developer can avoid passing it for the fields that doesnot interest them.

```swift
public init(metadata: [String: String]? = nil)

public init(userAttributes: [AuthUserAttribute]? = nil, metadata: [String: String]? = nil)

...
```

Reply:
fixed